monai/bundle/workflows.py

```diff
@@ -44,12 +44,18 @@ class BundleWorkflow(ABC):
         workflow_type: specifies the workflow type: "train" or "training" for a training workflow,
             or "infer", "inference", "eval", "evaluation" for a inference workflow,
             other unsupported string will raise a ValueError.
-            default to `train` for train workflow.
+            default to `None` for only using meta properties.
         workflow: specifies the workflow type: "train" or "training" for a training workflow,
             or "infer", "inference", "eval", "evaluation" for a inference workflow,
             other unsupported string will raise a ValueError.
             default to `None` for common workflow.
-        properties_path: the path to the JSON file of properties.
+        properties_path: the path to the JSON file of properties. If `workflow_type` is specified, properties will be
+            loaded from the file based on the provided `workflow_type` and meta. If no `workflow_type` is specified,
+            properties will default to loading from "train". If the specified file is unavailable, default properties
+            will be sourced from "monai/bundle/properties.py" based on the workflow_type:
+            For a training workflow, properties load from `TrainProperties` and `MetaProperties`.
+            For a inference workflow, properties load from `InferProperties` and `MetaProperties`.
+            For workflow_type = None : only `MetaProperties` will be loaded.
```

Comment:
I think if `properties_path` is not None and file is unavailable, we should raise error directly.
And if no `workflow_type` is specified, properties should not default to loading from "train", I would suggest the same behavior as `workflow_type` (default to `None` for only using meta properties)

Reply:
Hi @Nic-Ma, I have updated to load meta properties if workflow_type is None, please help review it again, thanks.

Reply:
As my previous comment, I think if properties_path is not None and file is unavailable, we should raise error directly. What do you think?

Thanks.

Reply:
Hi @Nic-Ma, yes. The logic is here:
https://github.com/Project-MONAI/MONAI/blob/ec202e0c75d6008326c055ac8df013405167ad29/monai/bundle/workflows.py#L116